contract/src/distribute.rs

```diff
@@ -1,119 +1,105 @@
-use aurora_launchpad_types::DistributionDirection;
+use aurora_launchpad_types::config::DistributionAccount;
 use near_plugins::{Pausable, pause};
+use near_sdk::json_types::U128;
 use near_sdk::serde_json::json;
 use near_sdk::{Gas, Promise, PromiseResult, env, near, require};
 
-use crate::traits::ext_ft;
 use crate::{
     AuroraLaunchpadContract, AuroraLaunchpadContractExt, GAS_FOR_FT_TRANSFER,
     GAS_FOR_FT_TRANSFER_CALL, ONE_YOCTO,
 };
 
 const GAS_FOR_FINISH_DISTRIBUTION: Gas = Gas::from_tgas(1);
```

Comment:
I doubt the loop to remove accounts from the distributed list can be reliably completed within 1 Tgas (previously only a single boolean value needed to be overwritten). I think this amount should be increased. Ideally it would be based on a benchmark ran on the real network, but my suggestion is 5 Tgas.